helpers/parameter_generator.py

```diff
@@ -189,7 +203,11 @@ def set_one_var_from_string(name, param_type, checks):
     return ret
 
 
-def gen_parameter_description(sections, descriptions, params_rst):
+def gen_parameter_description(
+    sections: List,
+    descriptions: List,
+    params_rst: Path
+):
```

Comment:
```suggestion
def gen_parameter_description(
    sections: List[Tuple[str, int]],
    descriptions: List[List[Dict[str, Any]]],
    params_rst: Path
) -> None:
```

Functions that return nothing should be annotated with `-> None`